include/libtorrent/peer_class.hpp

```diff
@@ -47,7 +47,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 namespace libtorrent
 {
-	typedef boost::uint8_t peer_class_t;
+	typedef boost::uint16_t peer_class_t;
```

Comment:
sure? because of having more then 65536 torrents with per torrent limit or what is the matter?